README.md

```diff
@@ -419,11 +419,11 @@ Based on [vim-easymotion](https://github.com/easymotion/vim-easymotion) and conf
 | vim.easymotionMarkerForegroundColorTwoChar | The font color for two-character markers, used to differentiate from one-character markers.                                                                                                                                                                       |
 | vim.easymotionMarkerWidthPerChar           | The width in pixels allotted to each character.                                                                                                                                                                                                                   |
-| vim.easymotionMarkerHeight                 | The height of the marker.                                                                                                                                                                                                                                         |
+| vim.easymotionDimBackground                | Whether to dim other text while markers are visible.                                                                                                                                                                                                              |
 | vim.easymotionMarkerFontFamily             | The font family used for the marker text.                                                                                                                                                                                                                         |
 | vim.easymotionMarkerFontSize               | The font size used for the marker text.                                                                                                                                                                                                                           |
 | vim.easymotionMarkerFontWeight             | The font weight used for the marker text.                                                                                                                                                                                                                         |
-| vim.easymotionMarkerYOffset                | The distance between the top of the marker and the text (will typically need some adjusting if height or font size have been changed).                                                                                                                            |
+| vim.easymotionMarkerMargin                 | Set the margin around a marker.                                                                                                                                                                                                                                   |
```

Comment:
Can you change this to read `The margin around each marker, in pixels.`? (and same in `package.json`)